webapp/src/index.tsx

```diff
@@ -5,14 +5,16 @@ import * as React from 'react';
 
 import {Channel} from 'mattermost-redux/types/channels';
 import {Post} from 'mattermost-redux/types/posts';
-
+import {getConfig} from 'mattermost-redux/selectors/entities/general';
 import Icon from './components/icon';
 import PostTypeJitsi from './components/post_type_jitsi';
 import I18nProvider from './components/i18n_provider';
 import RootPortal from './components/root_portal';
 import reducer from './reducers';
 import {startMeeting, loadConfig} from './actions';
 import {id as pluginId} from './manifest';
+import Client from './client';
+import {GlobalState} from 'mattermost-redux/types/store';
```

Comment:
Can we move this import up where the rest of the `mattermost-redux` imports are?

Reply:
@sanjaydemansol The below request is completed but this one still needs to be implemented

Reply:
@mickmister done